packages/p/pdfhummus/xmake.lua

```diff
@@ -37,5 +37,6 @@ package("pdfhummus")
     end)
 
     on_test(function (package)
-        assert(package:has_cincludes("PDFWriter.h"))
+        assert(package:has_cxxincludes("PDFWriter.h"))
+        assert(package:has_cxxtypes("PDFWriter"))
```

Comment:
please use has_cfuncs or has_cxxsnippets

Reply:
Still not right, see 
https://github.com/xmake-io/xmake-repo/blob/ce72a2f24f83a6e064ce6e0fdadca014006eacb3/packages/l/libtins/xmake.lua#L40

Reply:
PDFHummus has a test binary, but it is not installed, so I include a stub test in it.